packages/rn-tester/android/app/build.gradle

```diff
@@ -95,7 +95,7 @@ react {
 
 project.ext.react = [
     enableCodegen: true, // Keep this here until it's sync'ed to Android template.
-    enableFabric: (System.getenv('USE_FABRIC') ?: 'true').toBoolean()
+    enableFabric: (System.getenv('USE_FABRIC') ?: 'false').toBoolean()
```

Comment:
Generally RNTester assumes Fabric is true these days (this is also the case for iOS/tvOS, see the Podfile), so I think this should be reverted.

Reply:
Lol, I should stop creating PRs at 3 a.m.😂

Reply:
Obviously that's the best time to push code 😄